app.py

```diff
@@ -2,70 +2,112 @@
+def img(filepath):
+    return static_file(filepath, root="views/images")
 
-    print('username: ', vloga)
-    if not auth.obstaja_vloga(vloga):
-        return template("vloga.html", napaka="Vloga s tem imenom ne obstaja", url=url)
+@route("/static/<filename:path>", name="static")
+def static(filename):
+    return static_file(filename, root=static_dir)
 
-    response.set_cookie("vloga", vloga)
-    redirect(url('/'))
 
+# zacetna stran
+@get('/', name='index')
+def index():
+    znacka = preveriZnacko()
+    return template('zacetna_stran.html', url=url, znacka=znacka)
 
-@get('/odjava', name='odjava')
-def odjava():
-    """
-    Odjavi uporabnika iz aplikacije. Pobriše piškotke o uporabniku in njegovi vlogi.
-    """
-    response.delete_cookie("vloga")
-    return template('vloga.html', napaka=None, url=url, vloga=None)
+@get('/registracija', name='registracija_get')
+def registracija_get():
+    napaka = request.get_cookie("sporocilo")
+    vloga = request.get_cookie("vloga")
+    ime = request.get_cookie("ime")
+    priimek = request.get_cookie("priimek")
+    email = request.get_cookie("email", "_")
+    navijaska_drzava = request.get_cookie("navijaska_drzava")
+    return template("registracija.html", url=url, napaka=napaka, vloga=vloga, ime=ime, priimek=priimek, navijaska_drzava=navijaska_drzava, email=email)
+
+
+@post('/registracija', name='registracija_post')
+def registracija_post():
+    vloga = request.forms.vloga # type: ignore
+    ime = request.forms.ime # type: ignore
+    priimek = request.forms.priimek # type: ignore
+    email = request.forms.email # type: ignore
+    geslo = request.forms.geslo # type: ignore
+    geslo2 = request.forms.geslo2 # type: ignore
+    navijaska_drzava = request.forms.navijaska_drzava # type: ignore
+
+    registracija_ok_bool, msg = RegistracijaUtils().registracija_ok_q(
+        geslo=geslo, ponovljeno_geslo=geslo2, email=email, cur=cur
+        )
+    
+    if not registracija_ok_bool:
+        # nastavi_sporocilo(msg)
+        response.set_cookie("sporocilo", msg) #, secret=SECRET_COOKIE_KEY, path="/")
+        # imamo napako, nastavimo piškotke in preusmerimo
+        response.set_cookie('ime', ime, path="/", secret=SECRET_COOKIE_KEY)
+        response.set_cookie('priimek', priimek, path="/", secret=SECRET_COOKIE_KEY)
+        response.set_cookie('vloga', vloga, path="/", secret=SECRET_COOKIE_KEY)
+        response.set_cookie('navijaska_drzava', navijaska_drzava, path="/", secret=SECRET_COOKIE_KEY)
+        response.set_cookie('email', email, path="/", secret=SECRET_COOKIE_KEY)
+        redirect(url('registracija_get'))
+    
+    hash_gesla = RegistracijaUtils.izracunaj_hash_gesla(geslo)
+
+    try:
+        print("Vnos v bazo...")
+        cur.execute("""
+            INSERT INTO uporabniki (ime, priimek, email, geslo, navijaska_drzava)
+            VALUES (%s, %s, %s, %s, %s);
+        """, (ime, priimek, email, hash_gesla, navijaska_drzava))
```

Comment:
Na tej točki se mi zatakne... Kontekst funkcionalnosti bi moral biti razviden samo iz metode registracija_post.